spec/features/event/participation_spec.rb

```diff
@@ -49,18 +49,14 @@
   context "as admin" do
     before { sign_in(admin) }
 
-    it "creates invoice after accepting confirm alert" do
+    it "creates invoice when filling out form" do
```

Comment:
I tried adding feature specs to check that the price field is updated when changing reference_date or price_category, for some reason the Stimulus controller does not seem to be registered, other specs like `sac_terminate_membership_wizard_spec.rb` also use core stimulus controllers and work